src/ansys/dpf/post/post_utility.py

```diff
@@ -167,28 +182,43 @@ def load_simulation(
             )
             analysis_type = _AnalysisType.static
 
-    if physics_type == _PhysicsType.thermal:
-        if analysis_type == _AnalysisType.static:
-            raise NotImplementedError
-        elif analysis_type == _AnalysisType.transient:
-            raise NotImplementedError
-        else:
-            raise ValueError(f"Unknown analysis type '{analysis_type}' for thermal.")
-    elif (
-        physics_type == _PhysicsType.mecanic or physics_type == _PhysicsType.mechanical
-    ):
-        if analysis_type == _AnalysisType.static:
-            return MechanicalSimulation(data_sources, _model)
-        elif analysis_type == _AnalysisType.modal:
-            raise NotImplementedError
-        elif analysis_type == _AnalysisType.harmonic:
-            raise NotImplementedError
-        elif analysis_type == _AnalysisType.transient:
-            raise NotImplementedError
+        if physics_type == _PhysicsType.thermal:
+            if analysis_type == _AnalysisType.static:
+                raise NotImplementedError
+            elif analysis_type == _AnalysisType.transient:
+                raise NotImplementedError
+            else:
+                raise ValueError(
+                    f"Unknown analysis type '{analysis_type}' for thermal."
+                )
+        elif (
+            physics_type == _PhysicsType.mecanic
+            or physics_type == _PhysicsType.mechanical
+        ):
+            if analysis_type == _AnalysisType.static:
+                simulation_type = StaticMechanicalSimulation
+            elif analysis_type == _AnalysisType.modal:
+                simulation_type = ModalMechanicalSimulation
+            elif analysis_type == _AnalysisType.harmonic:
+                simulation_type = HarmonicMechanicalSimulation
+            elif analysis_type == _AnalysisType.transient:
+                simulation_type = TransientMechanicalSimulation
+            else:
```

Comment:
We should also allow comparison like:
```py
>>> analysis_type = "harmonic"
>>> analysis_type == _AnalysisType.static
False
```

Reply:
This could be done either overloading the "==" operator for the a new class AnalysisType:
```py
class AnalysisType:
     def __eq__(self, other):
           ....
_AnalysisType.static = AnalysisType("static")
assert analysis_type == _AnalysisType.static
assert "static" == _AnalysisType.static
```
 or just write the text comparison in the code:
```py
if analysis_type == _AnalysisType.static or analysis_type.lower() == "static":
    simulation_type = StaticMechanicalSimulation
elif analysis_type == _AnalysisType.modal or analysis_type.lower() == "modal":
    simulation_type = ModalMechanicalSimulation
elif analysis_type == _AnalysisType.harmonic or analysis_type.lower() == "harmonic":
    simulation_type = HarmonicMechanicalSimulation
elif analysis_type == _AnalysisType.transient or analysis_type.lower() == "transient":
    simulation_type = TransientMechanicalSimulation

else:
    raise ValueError(f"The analysis_type '{analysis_type}' is not accepted.")
```

Reply:
@germa89 actually the thing is we wanted to not expose _AnalysisType or _PhysicsType which come from pydpf-core, and instead give pre-canned strings to the user to define the exact combination they want (coresponging to available SimulationTypes).
We could accept string values, but these would not simply be "static" or "transient", these would be "static_mechanical" or "transient_mechanical". I feel like these give too many possibilities for errors and parsing them would not be robust.
What do you think?